modules/core/output_modules.php

```diff
@@ -650,7 +650,13 @@ class Hm_Output_loading_icon extends Hm_Output_Module {
      * Sort of ugly loading icon animated with js/css
      */
     protected function output() {
-        return '<div class="loading_icon"></div>';
```

Comment:
You renamed the class here: class="loading_bar" but it was refering to the module: `Hm_Output_loading_icon` you should keep it like that.